l1-contracts/contracts/bridgehub/STMDeploymentTracker.sol

```diff
@@ -25,6 +25,9 @@ contract STMDeploymentTracker is ISTMDeploymentTracker, ReentrancyGuard, Ownable
     /// @dev Bridgehub smart contract that is used to operate with L2 via asynchronous L2 <-> L1 communication.
     IL1AssetRouter public immutable override L1_ASSET_ROUTER;
 
+    /// @dev The encoding version of the data.
+    bytes1 internal constant ENCODING_VERSION = 0x01;
```

Comment:
we need to move the three encoding constants into some `common/Config` fiel

Reply:
They are different for each Bridge, so should they not be separate

Reply:
okay, makes sense then